wiki/Tournaments/GBC/GBC_2023_Autumn/en.md

```diff
@@ -0,0 +1,278 @@
+3. The final ranking of the Swiss round will be determined by the following criteria in order:
+   - Most matches won
+   - Highest beatmap difference (`beatmap wins - beatmap losses`)
+   - Highest Buchholz score[^bu]
+   - Head-to-head result (if possible)
+   - Most beatmaps won
+   - Fewest beatmaps lost
+   - Sum of all score differences of each beatmap (`∑(total score - opponent's total score)` for each beatmap played in the regular stage)
+   - Dice roll
+4. The top 16 teams will advanced to the playoff stage, while the remaining players get eliminated from the tournament.
+   - This means that teams with a record of 3-X will advance to the next stage, while teams with a record of X-3 will be eliminated.
+   - Also, teams with a record of 2-2 will have the chance to advance by ranking in the top 16.
+   - Teams with a record 3-0 or 0-3 need not play round 4, as they have already advanced or gotten eliminated.
+5. The results from the regular stage will also be used for seeding teams.
+
+### Win conditions
+
+- In the Qualifier stage, players need to rank high enough to advance.
+- In the drafting phase, players need to be picked into a team or become captains to advance.
+- In the regular stage, teams need to win 6 maps to win a match (best of 11).
+- In Quarterfinals, Semifinals, and Finals, teams need to win 7 maps to win a match (best of 13).
+
+### Match procedures
+
+1. A referee will create a multiplayer room 15 minutes before the scheduled match time. Players must join the lobby in that period.
+   - The room settings are `Game mode: "osu!mania"`, `Team mode: "Team Vs"` and `Score mode: "ScoreV2"`. The room name must follow the pattern of `GBC 2023 Autumn: ({TeamRed}) vs ({TeamBlue})`.
+   - The team mentioned first in the room name must be the red team, and the team mentioned second in the room name must be the blue team.
+2. Each captain can protect **one beatmap** from the pool. These beatmaps cannot be banned. Then, each captain can ban **one beatmap** from the pool. These beatmaps cannot be picked by any team for the entire duration of the match.
+3. Both captains will alternate picking a beatmap from the mappool.
+4. Each captain must use `!roll` once in `#multiplayer`.
+   - The winner of the `!roll` may choose to pick first or ban first in the match. The other choice will be used by loser of the roll. The team who picks first will also protect first, and the others will ban first.
+5. After each pick, both captains **may** ban **one** of the opposing team members from playing the beatmap in a match, **except during tiebreakers**.
+   - Any player can only be banned like this **once** per match.
+   - Only players who have already played in the match can be banned.
```

Comment:
Untranslated, is this rules changed?

Reply:
fixed on [3704f72](https://github.com/ppy/osu-wiki/pull/10075/commits/3704f7262ef29ea324bb6e985a23fd11d3930204)